plugins/modules/ovirt_disk.py

```diff
@@ -672,7 +673,7 @@ def main():
         host=dict(default=None),
         wipe_after_delete=dict(type='bool', default=None),
         activate=dict(default=None, type='bool'),
-        use_proxy=dict(default=False, type='bool'),
+        use_proxy=dict(default=None, type='bool'),
```

Comment:
Why change to None? use_proxy should be False by default.

Reply:
First I defaulted to not using a proxy which I guess would make sense for oVirt 4.4 upload directly to the node. But this would break existing oVirt 4.3 setups where the upload only works via proxy and direct node access is not possible (e.g. blocked via firewall).

Reply:
I had no issues when using `use_proxy: False` on oVirt 4.3. I was able to upload qcow and raw images without a problem.